tools/packaging/kernel/build-kernel.sh

```diff
@@ -144,7 +144,7 @@ get_kernel() {
 	[ -n "${kernel_path}" ] || die "kernel_path not provided"
 	[ ! -d "${kernel_path}" ] || die "kernel_path already exist"
 
-	if [ "${conf_guest}" != "" ]; then
+	if [[ "${conf_guest}" != "" && "${conf_guest}" != "sev" ]]; then
```

Comment:
I'm not sure if I understand this part here.  Why we would like to **not** get the kernel version from the `tee` entry in case of SEV?

Reply:
SEV can use the same base 5.19 kernel as general kata as all changes required for the specific kernel fork have been upstreamed. `get_tee_kernel` has some specifics regarding retrieving the version and associating it with the name of the tarball and the link to retrieve it. As kernel for SEV is the same as general kata now, I thought it best to have the script treat it the same. As an alternative, I can add handling in `get_tee_kernel` to fix the download link and the tarball name for SEV specifically.

Reply:
If SEV can be used from the upstream kernel, I'd recommend removing the SEV entry from the versions.yaml file.  The only reason it was added was because the upstream kernel didn't have all the needed patches.

Reply:
Hmm. Thinking a little bit more about it I'm not sure if we should have SEV support enabled by default on the guest kernel, thus having the separate path would still make sense.  I need to think a little bit more about this.

Reply:
So, no.  The kernel is the same, but the options for `SEV` are only being applied in case of `conf_guest` is set to `sev`, and I think we should keep it that way for the time being.

So, please, remove this part of the commit.

Reply:
I've removed this `conf_guest` check for `sev` and added a check in `get_tee_kernel` to see if `kernel_url` contains the string `cdn.kernel.org`. If it finds the generic kernel URL, it prepends `linux-` at the front or the tarball name. Tried to do this in the most generic way possible. Let me know if you'd like me to modify it differently.

Reply:
@ryansavino I've a suggestion to address the tarball name issue. I will comment in line.